server/config/config.js

```diff
@@ -0,0 +1,21 @@
+import dotenv from "dotenv";
+
+dotenv.config();
+
+module.exports = {
+  development: {
+    use_env_variable: "DATABASE_URL_DEVELOPMENT",
+    url: process.env.DATABASE_URL_DEVELOPMENT,
+    dialect: "postgres"
+  },
+  test: {
+    use_env_variable: "DATABASE_URL",
```

Comment:
Go and lookup doing tests with sqlite, They are quicker than postgres tests and do not require as much resources